src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -384,3 +413,21 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Account"
+    SubmitFilingPackageRequest:
+      description: submit filing package
+      type: object
+      required:
+        - isRushOrder
+      properties:
+        isRushOrder:
```

Comment:
Can you add a `acknowledge` date, this is to track when the thing was acknowledge